cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_cov.cpp

```diff
@@ -46,8 +46,8 @@ static train_result<Task> call_daal_kernel(const context_cpu& ctx,
     auto arr_data = row_accessor<const Float>{ data }.pull();
     auto arr_eigvec = array<Float>::empty(column_count * component_count);
     auto arr_eigval = array<Float>::empty(1 * component_count);
-    auto arr_means = array<Float>::empty(1 * component_count);
-    auto arr_vars = array<Float>::empty(1 * component_count);
+    auto arr_means = array<Float>::empty(1 * column_count);
+    auto arr_vars = array<Float>::empty(1 * column_count);
```

Comment:
Should we actually multiply it? As for me it doesn't provide any additional explicitness.
```suggestion
    auto arr_means = array<Float>::empty(column_count);
    auto arr_vars = array<Float>::empty(column_count);
```

Reply:
This helps to understand actual shape of the arr_vars, because it's used for `homogen_table` construction